### ISSUE TITLE
How to use it for Nextjs typescript ?

### ISSUE THREAD

**Issue by ender1102** [nghiepit/smooth-scrollbar-react#17]
Hello bro,

Can u help me use it for Nextjs, I have a issue. Thank for help
![image](https://user-images.githubusercontent.com/67373221/204071381-70d32b12-c34e-4652-b30e-889762b241b7.png)

**Comment by nghiepdev** [comment]
I have issue with Next.js 13, but [in Next.js 12](https://codesandbox.io/p/sandbox/hopeful-wozniak-d7wk6o?file=%2Fpages%2Findex.tsx&selection=%5B%7B%22endColumn%22%3A28%2C%22endLineNumber%22%3A41%2C%22startColumn%22%3A28%2C%22startLineNumber%22%3A41%7D%5D&workspace=%257B%2522activeFileId%2522%253A%2522cl8skx9ay000hlpge4cua49bz%2522%252C%2522openFiles%2522%253A%255B%2522%252Fpages%252Findex.tsx%2522%252C%2522%252Fpackage.json%2522%252C%2522%252Ftsconfig.json%2522%255D%252C%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522gitSidebarPanel%2522%253A%2522COMMIT%2522%252C%2522spaces%2522%253A%257B%2522claxlynwv000n3b6htn7krj4m%2522%253A%257B%2522key%2522%253A%2522claxlynwv000n3b6htn7krj4m%2522%252C%2522name%2522%253A%2522Default%2520Space%2522%252C%2522devtools%2522%253A%255B%257B%2522type%2522%253A%2522PREVIEW%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522key%2522%253A%2522claxqlz7y005r3b6iyig9ib6p%2522%252C%2522isMinimized%2522%253Afalse%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522key%2522%253A%2522claxqlx2s004b3b6in5z5yzgl%2522%252C%2522isMinimized%2522%253Afalse%257D%255D%257D%257D%252C%2522currentSpace%2522%253A%2522claxlynwv000n3b6htn7krj4m%2522%257D) works

**Comment by ender1102** [comment]
Yes, do you think you have to add "use client" for nextjs 13

**Comment by nghiepdev** [comment]
I don't think so, for the Next.js I usually use `next/dynamic with {ssr: false}` to resolve almost related issues SSR. Maybe `use client` for Next.js 13

**Comment by ender1102** [comment]
Well, I expect a new fix for nextjs typescript from you. Thank you
